src/configAPIClient.ts

```diff
@@ -1,13 +1,37 @@
-export default function ConfigAPIClient() {
-    this.getSDKConfiguration = function(
+import { DataPlanConfig } from '@mparticle/web-sdk';
+import { KitConfigs, MParticleWebSDK, SDKInitConfig } from './sdkRuntimeModels';
+
+export type SDKInitFunction = (
+    apiKey: string,
+    config: SDKInitConfig,
+    mpInstance: MParticleWebSDK
+) => void;
+
+export interface IConfigResponse {
+    appName: string;
```

Comment:
While this is sufficient to get your tests running, I'd make this config response more complete.  Here is an example.
https://jssdkcdns.mparticle.com/js/v2/beab3f4d34281d45bfcdbbd7eb21c083/config
(if you have a JSON reading plugin this makes it easier to read)
![image](https://user-images.githubusercontent.com/5377436/213229775-b434be70-e8a5-47ce-b0d7-687c1b1c7587.png)

So I'd suggest making this the full interface for the Config response to include the following as well:
* pixelConfigs: PixelConfig (see below comment on defining this)
* serviceUrl: string
* secureServiceUrl: string
* minWebviewBridgeVersion: number
* workspaceToken: string
* flags:
   * eventsV3: string
   * eventBatchingIntervalMillis: string
 * dataPlanResult: DataPlanResult (this is not part of the above screenshot because i don't have a dataplan, but I just slacked you [a link](https://mparticle.slack.com/archives/DHWS0FJTA/p1674060642699989) to see)

Also, add:
`export type PixelConfig = Dictionary` to `sdkrunTimemodels.ts` (right after `export type KitConfigs = Dictionary` on line 164) and import that here as the type

Reply:
I made some adjustments based on your feedback. Take a look and let me know if you think there's anything that should be optional rather than required or if we need to extend what I have.

We probably need to sync up on `DataPlanResult` but I think I'll put my notes in your comment where you directly reference it.